test/TritonGPU/amd/amd-stream-pipeline.mlir

```diff
@@ -0,0 +1,138 @@
+// RUN: triton-opt %s -split-input-file --tritonamdgpu-stream-pipeline | FileCheck %s
```

Comment:
This is too big. Can you reduce the test?

Reply:
I tried to do this. PTAL, thanks!